Makefile.PL

```diff
@@ -49,8 +49,8 @@ my %requires = (
     'Email::Send'              => '1.911',
     'File::Slurp'              => '9999.13',
     'JSON::XS'                 => '2.01',
-    'LWP::Protocol::connect'   => 0,
-    'LWP::UserAgent'           => '5.835',
+    'LWP::UserAgent'           => '6.26',
+    'LWP::Protocol::https'     => 6,
```

Comment:
I noted that these two lines switched order - but this list is sorted - so please swap the two.